wpilibNewCommands/src/main/java/edu/wpi/first/wpilibj2/command/Commands.java

```diff
@@ -166,6 +166,19 @@ public static Command either(Command onTrue, Command onFalse, BooleanSupplier se
     return new ConditionalCommand(onTrue, onFalse, selector);
   }
 
+  /**
+   * Runs a command if the boolean condition function is true otherwise it does nothing and exits.
```

Comment:
```suggestion
   * Runs a command if the boolean condition function is true. Otherwise it does nothing and exits
   * (but still has the same requirements as the command, interrupting any commands with shared
   * requirements).
```
I could see some people reading "does nothing" as "has no side effects" and being surprised/confused when it interrupts commands with shared requirements. Splitting the sentence is just personal preference that helps me parse it better, but go ahead and structure it how you think is best.